backend/typescript/services/implementations/campService.ts

```diff
@@ -537,24 +574,26 @@ class CampService implements ICampService {
 
         const campFee = newCamp.fee;
 
-        const dropoffPriceObject = await createStripePrice(
-          dropoffProductId,
-          dropoffFee * 100,
-        );
+        if (pickupFee && dropoffFee) {
```

Comment:
What about if only one if valid? It seems like this wouldn't handle just a single payment (ex. just pickupFee)

Reply:
there will always be both fees if either is present. This is some tech debt from before that I would love to clean up but essentially pickupFee is always the same as dropoffFee so idk why they are seperate fields. As such, if one is present, the other should be too